modules/README.md

```diff
@@ -0,0 +1,44 @@
+# How To
+This module is used to organize AWS SSO group and EKS RBAC Syncronization
+
+
+### Example 1
```

Comment:
Could you please provide example which works with EKS module? It needs to be clear to user how those should go along.